packages/sns/src/sns.wrapper.ts

```diff
@@ -374,6 +375,17 @@ export class SnsWrapper {
     params: Omit<QueryParams, "certified">
   ): Promise<GetStateResponse> => this.swap.state(this.mergeParams(params));
 
+  /**
+   * Returns the ticket if a ticket was found for the caller and the ticket
+   * was removed successfully. Returns None if no ticket was found for the caller.
+   * Only the owner of a ticket can remove it.
+   *
+   * Always certified
+   *
+   * @param params
+   */
+  notifyPaymentFailure = (): Promise<Ok_1> => this.swap.notifyPaymentFailure();
```

Comment:
Do you need to expose `Ok_1` and `NotifyPaymentFailureResponse` in `index.ts`?

Reply:
Yes, it would make more sense to return the ticket inside `Ok_1`